test/unit/transaction.test.js

```diff
@@ -985,42 +985,33 @@ describe('Transaction', function() {
     it('should create payload when DT enabled and CAT disabled', function() {
       tx.agent.config.cross_application_tracer.enabled = false
 
-      const payload = tx.createDistributedTracePayload().text()
+      const payload = tx._createDistributedTracePayload().text()
 
       expect(payload).to.not.be.null
       expect(payload).to.not.equal('')
     })
 
-    it('generates a priority for entry-point transactions', () => {
-      expect(tx.priority).to.equal(null)
-      expect(tx.sampled).to.equal(null)
-
-      tx.createDistributedTracePayload()
-
-      expect(tx.priority).to.be.a('number')
-      expect(tx.sampled).to.be.a('boolean')
-    })
-
     it('does not change existing priority', () => {
       tx.priority = 999
       tx.sampled = false
 
-      tx.createDistributedTracePayload()
+      tx._createDistributedTracePayload()
 
       expect(tx.priority).to.equal(999)
       expect(tx.sampled).to.be.false
     })
 
     it('sets the transaction as sampled if the trace is chosen', function() {
-      const payload = JSON.parse(tx.createDistributedTracePayload().text())
+      const payload = JSON.parse(tx._createDistributedTracePayload().text())
       expect(payload.d.sa).to.equal(tx.sampled)
       expect(payload.d.pr).to.equal(tx.priority)
     })
 
     it('adds the current span id as the parent span id', function() {
       agent.config.span_events.enabled = true
       agent.tracer.segment = tx.trace.root
-      const payload = JSON.parse(tx.createDistributedTracePayload().text())
+      tx.sampled = true
```

Comment:
I had to add `sample = true` here because before the "payload" method would call `_calculatePriority()`. And this would set `sampled` to `true` on the transaction. And when sampled was true, `createDistributedTracePayload()` [would add an `id` to the payload](https://github.com/newrelic/node-newrelic/blob/75729ded806c336d52bf685e43dbfa133c084b97/lib/transaction/index.js#L1205). Since `_calculatePriority()` was removed, the test below for `payload.d.id` would fail. So I had to manually add `sampled = true` to this test.